project/ProjectPlugin.scala

```diff
@@ -138,6 +140,17 @@ object ProjectPlugin extends AutoPlugin {
       )
     )
 
+    lazy val exampleRouteguideProtocolSettings: Seq[Def.Setting[_]] = Seq(
+      libraryDependencies ++= Seq(
```

Comment:
All `monix` would be needed here, the rest can be removed according to the comments above